_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Scrum Basics Series - Part 2: Scrum Roles"
+date: "2015-11-09 07:23:00"
+icon: university
+categories: agile
+authors: ["Anja B."]
+---
+
+Welcome to part two of our Scrum Basics Series.
+This part deals with the three different roles involved in the Scrum process.
+
+One thing to clarify right from the beginning: All three roles together form one Scrum Team.
+All are equal and neither the Product Owner (PO) nor the Scrum Master (SM) are superiors to the Development Team.
+To function well they need to work closely together and, at best, also sit in one room.
+They need to understand that they are ONE team, fighting for the same goal: to release the best possible product for the end-customer.
+
+# Development Team
+
+{% image blog-scrum2-devteam.jpg 50% left %}{% endimage %}
+
+A typical Development Team consists of 7 members +/-2 with cross-functional abilities.
+They should organise themselves and decide on their own, how tickets will be implemented within a sprint.
+It’s also the Development Team's decision, how many tickets they decide to solve within a sprint.
```

Comment:
It’s also the Development Team's decision on how many tickets they decide to solve within a sprint.